workflows/reusable_bump.yml

```diff
@@ -0,0 +1,77 @@
+
+name: "Bump"
+
+on:
+  workflow_call:
+    inputs:
+      force_version_bump:
+        required: false
+        default: ""
+        type: string        
+
+jobs:
```

Comment:
Should we have the `UnitTests` step here too? Looks to be missing when compared to the previous workflows

Reply:
No. If there were dependencies they would be specified in the workflow calling this reusable one.

For example:
```
  IntegrationTests:
      name: Mainline Integration Tests
      needs: TestMainline
      uses: aws-deadline/.github/workflows/reusable_integration_test.yml@mainline
      secrets: inherit
      with:
        repository: disposable-bealine-client-software
        branch: mainline
        environment: mainline

  Bump:
    name: Version Bump
    needs: IntegrationTests
    uses: aws-deadline/.github/workflows/reusable_bump.yml@mainline
    secrets: inherit
    with:
      force_version_bump: ${{ inputs.force_version_bump }}
```

Reply:
Thanks for the explanation!